Heine.Mvc.ActionFilters/Extensions/HttpContentExtensions.cs

```diff
@@ -44,6 +44,8 @@ string Format(string content)
                     case "application/xml":
                         try { return XDocument.Parse(content).ToString(); }
                         catch { return content; }
+                    case "application/pdf":
+                        return string.Empty;
```

Comment:
Instead of excluding certain mime types we should rather include the ones we are interested in, such as

- application/json
- appliaction/xml
- text/plain
- text/html

and return null or string.Empty in the default case. The list of unwanted mime types can be very long...

Do you agree @hfurubotten, @furier, @glennwiz?

Reply:
sounds reasonable to me, @hfurubotten what do you think?

Reply:
Its probably better to include the ones we know are useful to add in logging, instead of actually playing catch-up with new mime types that give odd results. 

Another idea is to make it configurable, and the user can maintain a list of mime types it should use the "default include" on (add it as a string to log message). Then we support the most common, xml and json, but also support more types if needed, and the rest not added from the user can be marked with a "Redacted" comment or similar.